packages/teleport/src/Desktops/Desktops.tsx

```diff
@@ -53,7 +60,7 @@ export function Desktops(props: State) {
         </Box>
       )}
       {attempt.status === 'failed' && <Danger>{attempt.statusText}</Danger>}
-      {attempt.status === 'success' && (
+      {!isEmpty && (
```

Comment:
the `isEmpty` flag will be true when it is still processing, so while processing it will probably show the table as well (in its empty state), so you'd probably want a `hasDesktops` flag